CONTRIBUTING.md

```diff
@@ -9,3 +9,42 @@ Please note we have a [code of conduct](https://openactive.io/public-openactive-
 Note that to set up a development environment for this project on a Mac, you will need:
 - Visual Studio for Mac, selecting only .NET Core features on setup - [Download](https://visualstudio.microsoft.com/vs/mac/)
 - .NET Core SDK 2.1.808 installer for Mac - [Download](https://dotnet.microsoft.com/download/dotnet-core/2.1)
+
+## Test Suite
+
+[**OpenActive Test Suite**](https://github.com/openactive/openactive-test-suite) is a suite of tests that can verify the conformance of an Open Booking API implementation, such as the **reference implementation** ([BookingSystem.AspNetCore](./Examples/BookingSystem.AspNetCore/)) contained within this project. OpenActive.Server.NET's CI checks that Test Suite passes for its reference implementation.
+
+## Pull Request Process
+
+Changes to [OpenActive.Server.Net](.) should be tested with the [Test Suite](#test-suite) before a Pull Request is submitted, to ensure that the reference implementation remains conformant. If changes are also required to Test Suite in order to properly test the new changes, then `coverage/*` branches should be used for both repositories, as documented in Test Suite's [Pull Request Process](https://github.com/openactive/openactive-test-suite/blob/master/CONTRIBUTING.md#pull-request-process) documentation.
```

Comment:
Link to here: https://app.gitbook.com/o/-LJTExWaXZFqcFJ_lZwF/s/bE7Sev9Msu9B0xnKKEpA/~/changes/5/reference-implementation-and-test-suite-overview/maintaining-test-suite-and-reference-implementation

When that's merged in?

Reply:
Thanks @civsiv . Totally agreed. I wasn't sure how to do it so I created a new draft issue for now: https://github.com/orgs/openactive/projects/17/views/8?pane=issue&itemId=39372185 and linked it to the issue for that Gitbook PR